src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -259,11 +264,27 @@
       (.setHost (:host config)))))
 
 (schema/defn ^:always-validate
+  queue-thread-pool :- QueuedThreadPool
+  [max-threads :- schema/Int
+   queue-max-size :- schema/Int]
+  (let [queue-min-size (if (> default-queue-min-threads queue-max-size)
+                         queue-max-size
+                         default-queue-min-threads)]
```

Comment:
Isn't this bit of logic equivalent to `(min default-queue-min-threads queue-max-size)`?